js/src/vector.ts

```diff
@@ -445,7 +445,7 @@ export function makeVector(init: any) {
             if (init instanceof DataView) {
                 init = new Uint8Array(init.buffer);
             }
-            const props = { offset: 0, length: init.length, nullCount: 0, data: init };
+            const props = { offset: 0, length: init.length, nullCount: -1, data: init };
```

Comment:
`_nullCount != 0 && (!nullBitmap || nullBitmap.byteLength == 0)` is not nullable so we set `-1` as the null count here to mark the data as not nullable. `makeData` will always create a null bitmap so I'm not sure why we need to check for no bitmap.